readme.md

```diff
@@ -2522,3 +2522,17 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 Contributions are welcome and encouraged! Read the [contribution guidelines][contrib-guide] first.
 
 [contrib-guide]: contributing.md
+
```

Comment:
If you look up a line or two, you can see the addition went too low, below the “contribute” paragraph. Please move your addition above this, preferably to one of the sections that makes sense, like “strategy”.